Jenkinsfile

```diff
@@ -161,10 +162,9 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
     ]
   )
 
-  // 
   pipeline.createStage(
```

Comment:
It seems the 2 docker build stages can run in parallel. Please let me know if the zLinux build takes too long and we need to convert them into parallel stages.